src/main/java/org/opentripplanner/model/StopTime.java

```diff
@@ -37,22 +37,38 @@
     /** This is a Conveyal extension to the GTFS spec to support Seattle on/off peak fares. */
     private String farePeriodId;
 
-    public StopTime() {
-
-    }
+    public StopTime() { }
 
     public StopTime(StopTime st) {
+        this.trip = st.trip;
+        this.stop = st.stop;
         this.arrivalTime = st.arrivalTime;
         this.departureTime = st.departureTime;
-        this.dropOffType = st.dropOffType;
-        this.pickupType = st.pickupType;
+        this.timepoint = st.timepoint;
+        this.stopSequence = st.stopSequence;
+        this.stopHeadsign = st.stopHeadsign;
         this.routeShortName = st.routeShortName;
+        this.pickupType = st.pickupType;
+        this.dropOffType = st.dropOffType;
         this.shapeDistTraveled = st.shapeDistTraveled;
-        this.stop = st.stop;
-        this.stopHeadsign = st.stopHeadsign;
-        this.stopSequence = st.stopSequence;
-        this.timepoint = st.timepoint;
-        this.trip = st.trip;
+        this.farePeriodId = st.farePeriodId;
+    }
```

Comment:
The above assignments are sorted in the same order as the declarations - making it easier to see which parameter witch might not be assigned.